ramls/item.json

```diff
@@ -57,7 +57,13 @@
       "properties": {
         "callNumber": {
           "type": "string",
-          "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item."
+          "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item.",
+          "readonly": true
+        },
+        "callNumberSuffix": {
```

Comment:
As this property is within the call number components object, does it need the leading `callNumber`? Are we envisaging other suffixes?

Reply:
I don't have a strong preference either way. @mreno-EBSCO @bohdan-suprun opinions?

Reply:
I'd prefer, in absence of conflict, that we remove redundant info in property names. I think `callNumberSuffix` is likely a candidate. I think `suffix` would suffice, but like you I can go either way since we've been referring to these things as `callNumberThis` and `callNumberThat` awhile. If I had to choose, I'd drop `callNumber` from these nested field names, well, obviously not for `callNumber` proper. 😄

Reply:
I'm going to close this PR as redundant. All the code is in #358 and it's easier to apply these updates to a single PR.